azure-pipelines.yml

```diff
@@ -0,0 +1,91 @@
+# Learn more: https://aka.ms/yaml
+
+variables:
+  DMD_BRANCH: $[ coalesce(variables['System.PullRequest.TargetBranch'], variables['Build.SourceBranchName'], 'master') ]
+
+jobs:
+  - job: Windows
+    timeoutInMinutes: 120
+    pool:
+      vmImage: 'vs2017-win2016'
+    variables:
+      VSINSTALLDIR: C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\
+      HOST_DMD_VERSION: 2.084.1
+    strategy:
+      matrix:
+        x64:
+          OS: Win_64
+          MODEL: 64
+          ARCH: x64
+          D_COMPILER: dmd
+    steps:
+      - script: set
+        displayName: environment
+      - script: systeminfo
+        displayName: systeminfo
+      - task: CopyFiles@2
+        inputs:
+          sourceFolder: '$(Build.SourcesDirectory)'
+          contents: '**/*'
+          TargetFolder: '$(Build.SourcesDirectory)/../druntime'
```

Comment:
You should not copy the phobos source to ../druntime ;-) Same below.

Reply:
Ah that was stupid. Thanks!